packages/suite/src/views/suite/notifications/index.tsx

```diff
@@ -33,7 +34,7 @@ const EmptyWrapper = styled.div`
     flex-direction: column;
     align-items: center;
     padding: 16px 32px 32px 32px;
-    height: 100vh;
+    height: ${isDesktop() ? `calc(100vh - ${DESKTOP_TITLEBAR_HEIGHT})` : '100vh'};
```

Comment:
hmm I think there shouldn't even be specified height for this component (it is vertically centering few elements that shouldn't really be centered, offset from the top would be much better), but ofc it's out of scope of this PR